tools/vagrant/bootstrap.sh

```diff
@@ -52,11 +52,7 @@ echo "export NRF52_SDK_ROOT=/usr/nrf52-sdk" >> ${HOME}/.bashrc
 
 sudo usermod -aG dialout vagrant
 
-# Make sure we're using Java 11 for Cooja
-sudo update-java-alternatives --auto
```

Comment:
Sorry for being unclear, I think this is still required, it's just the JAVA_HOME line that shouldn't be required.

Reply:
I tested out after removing the `sudo update-java-alternatives --auto` and it doesn't seem to be needed. The point is that only one version of java is installed under this revised script and the default is auto anyway. 

I should note that there are probably some other issues with the vagrant set-up - particularly around interaction with the vagrant-vbguest but that is probably for some other issue / pr.

Reply:
According to the ant documentation https://ant.apache.org/manual/Tasks/javac.html, the release parameter is ignored when compiling on JDK 8 or earlier.